setup.py

```diff
@@ -67,6 +67,10 @@ def get_git_sha() -> str:
         "sqlalchemy.dialects": [
             "postgres.psycopg2 = sqlalchemy.dialects.postgresql:dialect",
             "postgres = sqlalchemy.dialects.postgresql:dialect",
+            "superset = superset.extensions.metadb:SupersetAPSWDialect",
```

Comment:
Would `dialects` be a more appropriate name that `metadb`?

Reply:
Hmmm, good point. I'm not sure if we're ever going to have multiple SQLAlchemy dialects defined in Superset, which is why I named it this way.